src/Hazelcast.Net/Events/DistributedObjectLifecycleEventArgs.cs

```diff
@@ -19,7 +19,9 @@ namespace Hazelcast.Events
     /// <summary>
     /// Provides a base class for cluster object lifecycle events data.
     /// </summary>
+#pragma warning disable CA1711 // Identifiers should not have incorrect suffix - here it is correct
```

Comment:
Too many classes having this, maybe disable on assembly-level or in .csproj?

Reply:
yes - looking into it

Reply:
ok they _are_ already disabled at assembly level and these pragmas should not be here anymore, cleaning